firestore-tos-extension/PREINSTALL.md

```diff
@@ -0,0 +1,82 @@
+Use this extension to provides an API for adding new ackowledgement terms, retrieving the latest terms, and tracking acknowledgements.
+
+Allows users to sign ackowledgement forms as part of their application. Some common ackowledgement forms include ToS and cookie ackowledgement.
+
+## Usage
+
+The extension supports multiple use cases:
+
+- Accepting terms
+- Creating terms
+- Getting terms
+- Getting accepted terms
+
+All functions `require authentication` to run, ensure you have a valud Firebase User included in your POST requests.
+
+Managed documents are stored in a specified Cloud Firestore collection, while providing two subcollections for accesing data, these are:
+
+- acknowledgements
+- agreements
```

Comment:
lets call these notices not agreements

Reply:
Similar issue to naming above, should we reconsider what the sub-collections should be named as.

So far we have:

notices -> acknowledgments
notices -> agreements -> notices